hypothesis-python/src/hypothesis/internal/cache.py

```diff
@@ -31,6 +31,9 @@ def sort_key(self):
             return (1,)
 
 
+Unset = object()
+
```

Comment:
probably we could use `hypothesis.utils.conventions.not_set` instead? Not like there's any real difference between different sentinels, but may as well given that we have this convention

Reply:
Yep, thanks, didn't know about this. Will do!